omnigibson/utils/usd_utils.py

```diff
@@ -640,6 +592,33 @@ def reset(cls):
             cls.reset_raw_object_transforms_in_usd(prim)
         cls.MODIFIED_PRIMS = set()
 
+class SemanticsAPI:
+    """
+    Monolithic class for accessing Semantic information
+    """
+    _semantic_mapping = None
+
+    @classmethod
+    def get_class_mapping(cls):
+        """
+        Returns:
+            dict: Mapping from semantic class name (str) to semantic ID (int)
+        """
+        if cls._semantic_mapping is None:
+            cls._semantic_mapping = CLASS_NAME_TO_CLASS_ID
+        return cls._semantic_mapping
+
+    @classmethod
+    def get_semantic_mapping(cls):
+        """
+        Returns:
+            dict: Mapping from object's prim path (str) to its semantic ID (int)
+        """
+        paths_to_ids = dict()
+        for obj in og.sim.scene.objects:
+            paths_to_ids[obj.prim_path] = cls.get_class_mapping()[obj.category]
+        return paths_to_ids
```

Comment:
Do we need this? Looks like all it does is return the class name to class ID mapping

Reply:
I do agree we probably don't need this class for these two functions. But Josiah's branch also included functions to access global information for instance segmentation IDs. Do you think we still need that? 
On that note, is our current code already tracking instance IDs? If so, we can easily remap the replicator instance IDs over.

Reply:
For now, I just completely got rid of this class. Users can access this mapping using `CLASS_NAME_TO_CLASS_ID`. @cgokmen Do you think this is enough to establish the connection between `CLASS_NAME_TO_CLASS_ID` and semantic mapping? I personally think it would be a lot more intuitive if this can be named/aggregated with something that has more to do with semantics.